first-task/src/components/UserDetails.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import UserInfo from './UserInfo';
+import logo from '../logo.svg';
+
+export  default function UserDetails({user}){
+    return(
+        <div className="UserDetails">
+            <div className="Avatar">
+                <img src = {logo} alt="logo"/>
```

Comment:
I several times mentioned it that you don't need spaces before and after equal sign